docs/content/singleton-actors.md

```diff
@@ -0,0 +1,79 @@
+# Singleton Actors
+
+Singleton Actors allow a single source of truth for both operations and data across a multi-server simulation. They are server-side authoritative [Actors](https://docs.unrealengine.com/en-us/Programming/UnrealArchitecture/Actors) that are restricted to one instantiation on SpatialOS. Singleton Actors allow a single source of truth for both operations and data across a multi-server simulation. There are two kinds of Singleton Actors:
```

Comment:
"Singleton Actors allow a single source of truth for both operations and data across a multi-server simulation." twice?

Reply:
whoops